com.unity.perception/Runtime/GroundTruth/PerceptionCamera.cs

```diff
@@ -467,6 +467,25 @@ public bool RemoveLabeler(CameraLabeler cameraLabeler)
             return false;
         }
 
+        /// <summary>
+        /// Retrieves the given <see cref="CameraLabeler"/> from the list of labelers under this PerceptionCamera, if it
+        /// is in the list.  Returns null if not found
+        /// </summary>
+        /// <param name="cameraLabeler"></param>
+        /// <returns></returns>
+        public CameraLabeler GetLabeler<T>()
```

Comment:
Is this used anywhere? It's not bad to add, but I didn't include it before because it is valid to have multiple CameraLabelers of the same type.

Reply:
This was part of a dead-end in trying to get this all to work in the test, but ultimately I went with a test hook approach due to it's being a more direct way to get a given value into the shader and as such a more reliable test.  However, figured this might be useful functionality.  One approach would be to return a container instead of a single CameraLabeler - or do a string lookup to specify?  Alternatively, could just get rid of this - it's not doing anything in my changes at this point.

Reply:
I don't have a strong opinion either way. GameObject.GetComponent<T> has the same behavior, even though there can technically be more than one of the same type of component attached.

Reply:
I'll remove it for the time being - unclear what it'll be used for regardless.

Reply:
Not a huge fan of checking in code that isn't being used for anything and my introduce an unknown bug as a result if someone down the line actually does try to use the functionality to find that it wasn't built correctly.